### PR TITLE
Feature flex style

### PR DESCRIPTION
**Pull request by LEv145**
https://github.com/alemelis/sd-webui-ar/issues/15#issuecomment-1615870452
изображение

### CODE REVIEW

File: javascript/button_titles.js [hunk 1]
```diff
@@ -1,2 +0,0 @@
-// Do not put custom titles here. This file is overwritten each time the WebUI is started.
```
Comment (alemelis) [inline javascript/button_titles.js:1]: why was this file deleted?

**Review by alemelis — COMMENTED** [review]
thanks for the contribution!

just one question about `button_titles.js`, otherwise good to merge

File: javascript/button_titles.js (hunk 1 above)
Comment (LEv145) [reply javascript/button_titles.js:1]: The file is automatically created and changed, it was convenient for me to remove it from git control
